app/api/resources/user.py

```diff
@@ -276,10 +274,17 @@ def post(cls):
         if is_valid != {}:
             return is_valid, 400
 
-        result = DAO.create_user(data)
-
-        if result[1] is 200:
-            send_email_verification_message(data["name"], data["email"])
+        failed_send = ""
+        try:
+            send_email_verification_message(data['name'], data['email'])
+            
+        except:
+            failed_send = messages.REGISTRATION_FAILED
+        
+        if failed_send == "":
```

Comment:
Can be a boolean variable, if not that, at least we can use a boolean check as: if not failed_send rather than a string comparison.